frontend/src/styles/theme.css

```
@@ -0,0 +1,145 @@
+  --enterprise-bg: #121212;            /* Very Dark Grey/Black */
+  --enterprise-surface: #1e1e1e;       /* Dark Grey Surface */
+  --enterprise-text: #e9ecef;          /* Light Silver Text */
+  --enterprise-border: #2d2d2d;
+  --enterprise-accent: #6495ed;        /* Lighter Steel Blue for contrast */
+
+  /* Overriding Bootstrap Variables for Dark Mode */
+  --bs-body-bg: var(--enterprise-bg);
+  --bs-body-color: var(--enterprise-text);
+  --bs-primary: #e9ecef;               /* Silver/Platinum as primary in dark mode for contrast? Or keep it dark? */
+                                       /* Actually, let's keep Primary as the brand color, maybe a lighter charcoal or the accent */
+  --bs-primary: #4a6fa5;               /* Steel Blue */
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--bs-primary is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)